pages/test_analytics/swift_collectors.md.erb

```diff
@@ -12,15 +12,15 @@ You can also upload test results by importing [JSON](/docs/test-analytics/import
 
 Before you start, make sure XCTest runs with access to [CI environment variables](/docs/test-analytics/ci-environments).
 
-1. [Create a test suite](https://buildkite.com/docs/test-analytics), and copy the API token that it gives you.
+1. [Create a test suite](https://buildkite.com/docs/test-analytics), and copy the _Buildkite Test Analytics API token_ you see
 
-    To use the Buildkite Test Collector with a Swift Package Manager project, add this repository to the `Package.swift` manifest and add `BuildkiteTestCollector` to any test target requiring analytics:
+1. Add the `test-collector-swift` repository to the `Package.swift` manifest, and add `BuildkiteTestCollector` to any test target requiring analytics:
```

Comment:
I realize this didn't originate with this PR, but I wanted to flag something that this change made more apparent to me:

Not being a Swifter (Swifty?), it's not clear to me that this step happens entirely within the manifest file. I suspect this is actually two steps: adding `test-collector-swift` to `Package.swift` and modifying some other file (shown below) to add the collector to the targets. If so, might be nice to put the `Package.swift` bit on its own step.

Reply:
Me either, but I don't think so. I think **both of these** are in Package.swift, just two sections, *dependencies* and *targets*